### ISSUE TITLE
npm install unmet peer dependency catch22

### ISSUE THREAD

**Issue by dyspop** [leemunroe/grunt-email-workflow#96]
When I unstall grunt-email-workflow:
```
$npm install
[ommitted output]
$npm ls
[ommitted output]
npm ERR! peer dep missing: grunt@^1.0.1, required by grunt-mailgun@2.0.1
npm ERR! peer dep missing: grunt@^1.0.1, required by grunt-mailgun@2.0.1
$npm install grunt@1.0.1
[omitted output]
$npm ls
npm ERR! peer dep missing: grunt@~0.4.2, required by grunt-autoprefixer@2.2.0
npm ERR! peer dep missing: grunt@~0.4.0, required by grunt-aws-s3@0.13.1
npm ERR! peer dep missing: grunt@~0.4.1, required by grunt-cloudfiles@0.3.0
npm ERR! peer dep missing: grunt@~0.4.0, required by grunt-contrib-clean@0.6.0
npm ERR! peer dep missing: grunt@~0.4.0, required by grunt-contrib-watch@0.6.1
npm ERR! peer dep missing: grunt@~0.4.0, required by grunt-express@1.4.1
npm ERR! peer dep missing: grunt@^0.4.5, required by grunt-juice-email@0.1.4
npm ERR! peer dep missing: grunt@~0.4.2, required by grunt-litmus@0.1.8
npm ERR! peer dep missing: grunt@~0.4.0, required by grunt-contrib-watch@0.6.1
npm ERR! peer dep missing: grunt@~0.4.0, required by grunt-parallel@0.3.1
``` 

tl;dr: grunt-mailgun requires newer grunt, a bunch of other modules require old grunt.

**Comment by leemunroe** [comment]
@dyspop Dan thanks for the heads up. I'll take a look. If you determine what the best version is to use in the mean time, for grunt-mailgun and others, feel free to commit back to the repo.

**Comment by leemunroe** [comment]
All looks good to me now. Closing.
